ttyplot.c

```diff
@@ -610,7 +610,13 @@ int main(int argc, char *argv[]) {
             if (tty >= select_nfds)
                 select_nfds = tty + 1;
         }
-        struct timespec timeout = { .tv_sec = 0, .tv_nsec = 500e6 };  // 500 milliseconds for refreshing the clock
+        // Refresh the clock on the next full second.
+        static const int nanoseconds_per_second = 1e9;
```

Comment:
The `static` seems unnecessary here. I haven't checked whether it does damage, but it's not needed, my vote for dropping. Ideally, the compiler optimizes away all of it and then it doesn't use memory.

Reply:
Your are right, `static` does not bring us anything here. I'm with you: let's trust the compiler, at least for the very basic optimizations like this. Updated.